culture/review_culture.md

```diff
@@ -0,0 +1,142 @@
+> If you want to go fast, go alone. If you want to go far, go together.
+
+As a community, we value going far over going fast. To go far, we need to stay
+aligned in our values, goals, and practices. One of the ways that we do this is
+through cultivating review culture.
```

Comment:
This is a hugely annoying detail (and something I don't tend to do for the repos I maintain) but I've seen it in various communities so I want to mention. Should there be standards around how to write the markdown? For example, I've worked with folks that like to have one sentence per line, because it makes editing / commenting / seeing changes easier on GitHub.

Reply:
I would like to have standards around markdown, and I think they are out-of-scope for a guide on review culture in general. They would, however, be very relevant as contributing guidelines for this repository (sourcecred/docs) in particular. I made an issue on this repo to discuss: https://github.com/sourcecred/docs/issues/3